src/components/WalletStatementModal/index.native.tsx

```diff
@@ -1,31 +1,29 @@
-import lodashGet from 'lodash/get';
 import React, {useCallback, useRef} from 'react';
 import {withOnyx} from 'react-native-onyx';
-import {WebView} from 'react-native-webview';
-import _ from 'underscore';
+import {WebView, WebViewNavigation} from 'react-native-webview';
+import {ValueOf} from 'type-fest';
 import FullScreenLoadingIndicator from '@components/FullscreenLoadingIndicator';
 import Navigation from '@libs/Navigation/Navigation';
 import * as Report from '@userActions/Report';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
-import {walletStatementDefaultProps, walletStatementPropTypes} from './WalletStatementModalPropTypes';
+import type {WalletStatementOnyxProps, WalletStatementProps} from './types';
+
+type WebViewNavigationEvent = WebViewNavigation & {type?: ValueOf<typeof CONST.WALLET.WEB_MESSAGE_TYPE>};
```

Comment:
Interesting. Do you think that we might benefit from making `CONST.js` more TypeScript-friendly? It looks like we might find some uses for TypeScript enums.

Reply:
Still, the question holds. While I appreciate the `ValueOf` magic, I'm wondering if making things more TypeScriptish on the `CONST.ts` side could help with making this less magical.

Reply:
@VickyStash You could extract `ValueOf<typeof CONST.WALLET.WEB_MESSAGE_TYPE>` to a separate type.

Reply:
@cubuspl42 It's something that goes from [TS_STYLE](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#enum-vs-union-type) guide, so I'm not sure that we should change it.

Reply:
> Still, the question holds. While I appreciate the ValueOf magic, I'm wondering if making things more TypeScriptish on the CONST.ts side could help with making this less magical.

Once the codebase is completely written in TS, we can rewrite CONST.ts more safely, for now let's keep it as it is.

Reply:
Fine 👍